File: python/oper/Operation.py
def calc():
    name = "Andrei"
    last = "MMM"

    print(f"{name} {last} age ?")

    age = 0
    setedAge = False
    while not setedAge:
        try:
            age = int(input("Get Age?"))
            setedAge = True
        except ValueError:
            print("Wrong int:")

    print(f"Varsta mea la anu:{age + 1}")

File: cpp/oper/Operation.cpp
//
// Created by compandreimd on 19.05.2024.
//

#include "Operation.hpp"
#include <iostream>

void oper::Operation::calc() {
    const char *name="Andrei";
    const char *last= "MMM";

    std::cout <<  name << " " << last << " age ?";

    unsigned int age = 0;
//    bool setedAge = false;  !!ERROR INSERT NOT§ NUMBER
//    while (!setedAge) {
       // try {
            std::cin >> age;
        //    setedAge = true;
//        } catch (int a) {
//            std::cout << "Wrong int:";
//        }
//    }

    std::cout << "Varsta mea la anu:"  <<  (age + 1);

}